src/pages/case-studies/[slug].astro

```diff
@@ -0,0 +1,175 @@
+---
```

Comment:
Are there many differences between this design and the blog post layout?  If they're pretty close it might be worth sharing a `ProseLayout.astro` or similar with most of the logic for cover images, post headers, etc

Reply:
I had the same thought, but I didn't want to abstract eagerly

Reply:
The layout ended up changing in _lots_ of ways, so I'll keep it duplicated 😇